src/wyzeapy/services/bulb_service.py

```diff
@@ -112,17 +115,25 @@ async def turn_on(self, bulb: Bulb, local_control, options=None):
         if options is not None:
             plist.extend(options)
 
-        if bulb.type in [
-            DeviceTypes.LIGHT
-        ]:
+        if bulb.type is DeviceTypes.LIGHT:
             await self._set_property_list(bulb, plist)
+
         elif (
             bulb.type in [DeviceTypes.MESH_LIGHT, DeviceTypes.LIGHTSTRIP]
         ):
+            # Local Control
             if local_control and not bulb.cloud_fallback:
                 await self._local_bulb_command(bulb, plist)
-            else:
+
+            # Cloud Control
+            elif bulb.type is DeviceTypes.MESH_LIGHT:  # Sun match for mesh bulbs needs to be set on a different endpoint for some reason
+                for item in plist:
```

Comment:
Why can't we just send all the values to the _set_property_list endpoint at once?

Reply:
Good question.. The mesh bulbs only seem to work with `_run_action_list`, except for when I try to change the sun match value. If I send that with the rest of the commands to `_run_action_list` nothing happens. Same is true if I try and send any other values via `_set_property_list`. So the only way I can get it to work on these bulbs is to send the sun match value to a separate endpoint. The good news is this is only used when not in local control, so it probably wont come up much. I only randomly stumbled on it while testing cloud control.